src/shared/components/query/QueryStore.ts

```diff
@@ -247,6 +247,16 @@ export class QueryStore {
         return referenceGenomes.length === 1;
     }
 
+    @computed
+    get onlyOneReferenceGenomeInAllStudies() {
```

Comment:
can we reverse the sign of this to multipleReferenceGenomesPresentInAllStudies?

Reply:
fixed in [765125b](https://github.com/cBioPortal/cbioportal-frontend/pull/4407/commits/765125bb1ba294e20e059cfe213bce3d7712325b)